0001-basic-expiry-model.md

```diff
@@ -0,0 +1,160 @@
+This is, a function returning the minimum time taken for <img src="https://latex.codecogs.com/gif.latex?C_{k}"/> to take place with probability <img src="https://latex.codecogs.com/gif.latex?p"/>.
+
+Quantile functions for Erlang random variables can only be approximated via [numerical methods](https://docs.scipy.org/doc/scipy-0.16.1/reference/generated/scipy.stats.erlang.html).
+
+Knowing all this, `alpha_expiry` can be estimated:
+
+<img src="https://latex.codecogs.com/gif.latex?E_{\alpha}=E_{\beta}&plus;T_{\alpha}&plus;Q_{C_{k}}(p)"/>,
+
+where the value of <img src="https://latex.codecogs.com/gif.latex?p"/> is directly proportional to the security of the model (and the size of `alpha_expiry`).
+
+### Beta expiry
+
+There are two ways in which `beta_expiry`, <img src="https://latex.codecogs.com/gif.latex?E_{\beta}"/>, has an effect on the protocol:
+
+1. How long Alice will (at least) have to redeem `beta_asset` if Bob funds `beta_asset`.
+2. How long Bob will have to wait to be able to refund `beta_asset` if Alice has yet to redeem it.
+
+Consequently, it would be ideal to find a value of `beta_expiry` that lets Alice redeem `beta_asset` with high probability, allowing the protocol to continue, while also ensuring Bob doesn't have to wait too long to be able to be refunded if it comes to that.
+
+#### Assumptions
+
+- Alice provides <img src="https://latex.codecogs.com/gif.latex?T^{\alpha}_{A}"/>, the time in which she is overwhelmingly confident she will be able to confirm a transaction on `alpha_ledger`.
+She has direct influence over this since she can choose the fee.
+- Bob provides <img src="https://latex.codecogs.com/gif.latex?T^{\beta}_{B}"/>, the time in which he is overwhelmingly confident he will be able to confirm a transaction on `beta_ledger`.
+He has direct influence over this since he can choose the fee.
+- Alice provides <img src="https://latex.codecogs.com/gif.latex?T^{\beta}_{A}"/>, the time in which she is overwhelmingly confident she will be able to confirm a transaction on `beta_ledger`.
+She has direct influence over this since she can choose the fee.
+- Alice provides <img src="https://latex.codecogs.com/gif.latex?k"/>, the number of additional confirmations she would need to have on her redeem transaction before she considers `beta_asset` to be her own.
+- Block time on `beta_ledger` can be modelled as a random variable that follows an exponential distribution with parameter <img src="https://latex.codecogs.com/gif.latex?\lambda"/>.
+- A confirmation is given by the arrival of a new block.
+
+#### Model
+
+To determine `beta_expiry`, the random variable <img src="https://latex.codecogs.com/gif.latex?R_{\beta}"/>, which models the time Alice will need to get her desired number of confirmations on her redeem transaction of `beta_asset`, will need to be approximated.
```

Comment:
🔧 This was too much passive voice for me
```suggestion
To determine `beta_expiry`, we need to approximate the random variable <img src="https://latex.codecogs.com/gif.latex?R_{\beta}"/>, the time Alice will need to get her desired number of confirmations on her redeem transaction of `beta_asset`
```

Reply:
We may disagree on this, but I generally prefer the passive voice for maths. Since I wrote it in the passive voice already, I'd like to keep it consistent.